mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -196,8 +196,21 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_24833 getBoundingBox (Lnet/minecraft/class_4050;)Lnet/minecraft/class_238;
 		ARG 1 pose
 	METHOD method_25937 displaySoulSpeedEffects ()V
-	METHOD method_26082 applyStatusEffect (Lnet/minecraft/class_1293;Lnet/minecraft/class_1297;)V
+	METHOD method_26082 setStatusEffect (Lnet/minecraft/class_1293;Lnet/minecraft/class_1297;)V
+		COMMENT Sets a status effect in this entity.
+		COMMENT
+		COMMENT <p>This replaces the preexistent status effect of the same type on this entity.
```

Comment:
```suggestion
		COMMENT <p>This replaces the pre-existing status effect of the same type on this entity.
```

Reply:
I'm fairly sure it is a word. https://www.dictionary.com/browse/preexistent

Reply:
Would this be better?

```suggestion
		COMMENT <p>If this entity has a status effect of the same type already, it is replaced.
```

I'm not sure if status effect priority rules will apply here.

Reply:
```suggestion
		COMMENT <p>The preexistent status effect of the same type on this entity, if there is one, is cleared.
```